inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -169,72 +223,129 @@
+$if(authors.thanks)$\corresp[\fnsymbol{thisauththanks}]{$authors.thanks$}$endif$
+
 $endfor$
-%
-% Add \and if both authors and author
-$if(authors)$$if(author)$\and$endif$$endif$%
-%
-% Code for new (elsevier) style author field
+
 $for(author)$
-\name{$author.name$}
-$if(author.affiliation)$
+
+\author[%
+$for(author.affiliation)$\addrlblbycode{$author.affiliation$}$sep$,$endfor$%
+$if(author.corresponding_author)$$if(author.affiliation)$,$endif$\refstepcounter{correspcnt}\setcounter{thisauthcorresp}{\value{correspcnt}}\fnsymbol{thisauthcorresp}$endif$%
+$if(author.thanks)$$if(author.affiliation)$,$elseif(author.corresponding_author)$,$endif$\refstepcounter{correspcnt}\setcounter{thisauththanks}{\value{correspcnt}}\fnsymbol{thisauththanks}$endif$%
+$if(author.footnote)$$if(author.affiliation)$,$elseif(author.corresponding_author)$,$elseif(author.thanks)$,$endif$$for(author.footnote)$\corresplblbycode{$author.footnote$}$sep$,$endfor$$endif$%
+]{$author.name$}
+
 $for(author.affiliation)$
-\address{\Ref{ADR}{$author.affiliation$}}
+\addrbycode{$author.affiliation$}
 $endfor$
-$endif$%
-$if(author.email)$\email{\href{mailto:$author.email$}{$author.email$}}$endif$%
-$if(author.corresponding_author)$\thanks{Corresponding author; Email: \href{mailto:$author.email$}{$author.email$}}$endif$%
-$if(author.thanks)$\thanks{$author.thanks$}$endif$%
-$if(author.footnote)$
+
+$if(author.corresponding_author)$\corresp[\fnsymbol{thisauthcorresp}]{Corresponding author. \href{mailto:$author.email$}{\nolinkurl{$author.email$}}}$endif$
+$if(author.thanks)$\corresp[\fnsymbol{thisauththanks}]{$author.thanks$}$endif$
+
 $for(author.footnote)$
-\thanksbycode{$author.footnote$}
+\correspbycode{$author.footnote$}
 $endfor$
-$endif$%
-$sep$\and
+
 $endfor$
-%
-}
 
-\abstract{$abstract$}
+% Add author mark
+$if(authormark)$
+\authormark{$authormark$}
+$elseif(author)$
+\authormark{$for(author/first)$$it.name$$endfor$$if(author/rest/rest)$ et al.$else$$for(author/rest)$ \and $author.name$$endfor$$endif$}
+$elseif(authors)$
+\authormark{$for(authors/first)$$it.name$$endfor$$if(authors/rest/rest)$ et al.$else$$for(authors/rest)$ \and $authors.name$$endfor$$endif$}
+$endif$
+
+\received{Date}{0}{Year}
+\revised{Date}{0}{Year}
+\accepted{Date}{0}{Year}
 
-\date{$if(date)$$date$$else$\today$endif$}
+%\editor{Associate Editor: Name}
+
+\abstract{
+$for(abstract)$$abstract$$sep$\\
+$endfor$}
 
 \keywords{$for(keywords)$$keywords$$sep$; $endfor$}
 
-\maketitle
+$if(boxedtext)$
+\boxedtext{
+$boxedtext$}
+$elseif(boxedlist)$
+\boxedtext{
+\begin{itemize}
+$for(boxedtext)$
+\item $boxedtext$
+$endfor$
+\end{itemize}}
+$endif$
 
+\maketitle
 
 $for(include-before)$
 $include-before$
 $endfor$
 
 $body$
 
-$if(notes)$
-\begin{notes}
-$notes$
-\end{notes}
+%%%%%%%%%%%%%%
+
+%\begin{appendices}
+%\end{appendices}
+
+$if(competing_interests)$
+\section{Competing interests}
+
+$competing_interests$
 $endif$
```

Comment:
This could also be but in end of body as usual markdown and the other below also. 
Anything before bibliography insertation can be done this way. 

````
# Competing interests

There is NO Competing Interest

# Author contributions statement

Must include all authors, identified by initials, for example:
S.R. and D.A. conceived the experiment(s),  S.R. conducted the experiment(s), S.R. and D.A. analysed the results.  S.R. and D.A. wrote and reviewed the manuscript.

# Acknowledgments

The authors thank the anonymous reviewers for their valuable suggestions. This work is supported in part by funds from the National Science Foundation (NSF: \# 1636933 and \# 1920920).
````

But using YAML header is also fine.

Reply:
I think I did this as other formats did this and these sections seemed special and I was worried that the CLS might do something special with these sections. It is also information that may be excluded for double-blind reviewing, so somewhat convenient to have it all in the YAML header.

Reply:
>  think I did this as other formats did this and these sections seemed special and I was worried that the CLS might do something special with these sections

Some templates wants this in places that are not adapted to being at the end of the body. Hence the YAML trick to make them appear in specific place in the template, outside of body. 

> It is also information that may be excluded for double-blind reviewing, so somewhat convenient to have it all in the YAML header.

This is ok to me. Let's keep it that way !